csi/moac/node.js

```diff
@@ -88,6 +88,12 @@ class Node extends EventEmitter {
     this._offline();
   }
 
+  unbind () {
+    // todo: on graceful removal do we need to destroy the pools as well?
+    this.pools.forEach((pool) => pool.unbind());
```

Comment:
IMHO, no never. Deleting data is not something we should be doing. Who knows what (a) type of data it is and (b) what else the devices might be used for, right?

Reply:
Yeah we should only delete user data when we're really sure it's safe to do so. When a user removes a node explicitly (better wording than gracefully) is a potential candidate for it.